### ISSUE TITLE
spatial loading via GUI is broken

### ISSUE THREAD

**Issue by bjhecht** [shilad/wikibrain#230]
hardcoded username, some bug with password, dbname not correctly written to customized.conf, etc.

**Comment by shilad** [comment]
I've tested, but not in the last few releases.

That stuff *should* all be written correctly as long as you've selected
Postgres as your main database. Could you attach the relevant
customized.conf?

**Comment by bjhecht** [comment]
Screenshot and conf file attached :) Thanks! Getting back into the swing of things here :)

![screen shot 2015-03-08 at 1 39 43 pm](https://cloud.githubusercontent.com/assets/1738641/6547026/9b2e7144-c598-11e4-8a86-037769528284.png)

--- conf file ---
baseDir : "\/wikibrain"

dao.dataSource.default : psql

dao.dataSource.h2.url : "jdbc:h2:"${baseDir}"\/db\/h2;LOG=0;CACHE_SIZE=65536;LOCK_MODE=0;UNDO_LOG=0;MAX_OPERATION_MEMORY=100000000"

dao.dataSource.psql.url : "jdbc:postgresql:\/\/localhost\/wikibrain_spatial"
dao.dataSource.psql.username : "bjhecht"
dao.dataSource.psql.password : "[C@35d8cf6d"

dao.dataSource.postgis.host : "localhost"
dao.dataSource.postgis.port : "5432"
dao.dataSource.postgis.database : "wikibrain_spatial"
dao.dataSource.postgis.user : "bjhecht"
dao.dataSource.postgis.passwd : "[C@4d3bbe9c"

**Comment by bjhecht** [comment]
Also, this occurs with 'h2' as the database as well. Perhaps we should force Postgres if they choose spatial?

**Comment by shilad** [comment]
Good idea. Could you create a separate bug for that, and rename this one to
be more specifically about PG? What happens if you add a fake password?
Everything looks good in the configuration except for the password field.
